### PR TITLE
[Hub Generated] Review request for Microsoft.CostManagement to add version stable/2020-06-01

### PR DESCRIPTION
**Pull request by Zzegfried**
This is a PR generated at [OpenAPI Hub](https://aka.ms/openapihub). You can view your [work branch via this link](https://portal.azure-devex-tools.com/branch/melmullen/azure-rest-api-specs/dev-cost-management-Microsoft.CostManagement-2020-06-01...master/).

### Contribution checklist:
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for AME onboarding.
- [x] [Validation tools](https://aka.ms/swaggertools) have been run on swagger spec(s) updated and errors have all been fixed with this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, [please view the FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist

- [x] Service team MUST add the "**WaitForARMFeedback**" label if the management plane API changes fall into one of the below categories.
*	adding/removing APIs.
* adding/removing properties.
* adding/removing API-version.
* adding a new service in Azure.

*Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.*

- [x] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them.

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

File: specification/cost-management/resource-manager/readme.md [hunk 1]
```diff
@@ -26,18 +26,30 @@ These are the global settings for the Cost Management API.
 
 ``` yaml
 openapi-type: arm
-tag: package-2019-11
-azure-validator: false
+tag: package-2020-06
+azure-validator: true
```
Comment (qiaozha) [inline specification/cost-management/resource-manager/readme.md:30]: I remember there's an issue in modelerfour about this if azure-validator set as true. That's why the azure-cli-extension automation task fails.

File: specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2020-06-01/costmanagement.json [hunk 2]
```diff
@@ -0,0 +1,3287 @@
...
+        },
+        "fileName": {
+          "description": "The name of the exported file.",
+          "type": "string"
+        },
+        "runSettings": {
+          "description": "The export settings that were in effect for this execution.",
+          "$ref": "#/definitions/CommonExportProperties"
+        },
+        "errorCode": {
```
Comment (pilor) [inline specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2020-06-01/costmanagement.json:3127]: please place "code" and "message" inside an "error" object so it conforms to the standard error structure

**Review by pilor — CHANGES_REQUESTED** [review]

File: specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2020-06-01/costmanagement.json (hunk 2 above)
Comment (melmullen) [reply specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2020-06-01/costmanagement.json]: Done

**Review by pilor — APPROVED** [review]

**Pull request merged**